5e-SRD-Subclasses.json

```diff
@@ -49,7 +49,7 @@
 			"type": "level"
 		}],
 		"spell": {
-			"url": "/api/spells/28",
+			"url": "/api/spells/blade-barrier",
```

Comment:
Cool. All of the spells in this file had the  wrong spell id. Good thing we're doing this.